src/daml/Daml/Trading/Account/Account.daml

```diff
@@ -138,6 +149,16 @@ template Account
         
         create this with optNftAccount = Some (accountFactoryCid, holdingFactoryCid, accountKey)
     
+    choice Create_Badge_Account: ContractId Account
```

Comment:
Badge is an item not an Account, please change choice name

Reply:
I've already removed this choice and created Redeem Badges choice in Account template